options/posix/generic/sys-time-stubs.cpp

```diff
@@ -68,3 +68,8 @@ int setitimer(int which, const struct itimerval *new_value, struct itimerval *ol
 	}
 	return 0;
 }
+
+int timer_create(clockid_t, struct sigevent *__restrict, timer_t *__restrict) {
+	__ensure(!"Not implemented");
+	__builtin_unreachable();
```

Comment:
Didn't we have an implementation of this in some other PR (and can we add one if we didn't?)

Reply:
Not that I would know of or could find. Given that the implementation is non-trivial and I wouldn't know anything that uses it, I'd leave it for a future PR.